pkg/kv/store_message_test.go

```diff
@@ -25,27 +25,24 @@ func TestStoreMessage(t *testing.T) {
 	ctx := context.Background()
 	store := kvtest.GetStore(ctx, t)
 
-	sm := kv.StoreMessage{
-		Store: store,
-	}
 	t.Run("set-get test", func(t *testing.T) {
-		testStoreMessageSetGet(t, ctx, sm)
+		testStoreMessageSetGet(t, ctx, store)
 	})
 	t.Run("set-if test", func(t *testing.T) {
-		testStoreMessageSetIf(t, ctx, sm)
+		testStoreMessageSetIf(t, ctx, store)
 	})
 	t.Run("delete test", func(t *testing.T) {
-		testStoreMessageDelete(t, ctx, sm)
+		testStoreMessageDelete(t, ctx, store)
```

Comment:
Redundant - we have per driver tests for this operation

Reply:
Removed - thanks!